--- a/main/java/net/minecraft/server/ServerAdvancementManager.java
+++ b/main/java/net/minecraft/server/ServerAdvancementManager.java
@@ -41,7 +41,7 @@
             Advancement.Builder advancement$builder = Advancement.Builder.m_138380_(jsonobject, new DeserializationContext(p_136039_, this.f_136024_));
             map.put(p_136039_, advancement$builder);
          } catch (IllegalArgumentException | JsonParseException jsonparseexception) {
-            f_136021_.error("Parsing error loading custom advancement {}: {}", p_136039_, jsonparseexception.getMessage());
+            f_136021_.error("カスタムアドバンスの読み込みエラーの解析 {}: {}", p_136039_, jsonparseexception.getMessage());
          }
 
       });
