--- a/main/java/net/minecraft/nbt/NbtAccounter.java
+++ b/main/java/net/minecraft/nbt/NbtAccounter.java
@@ -15,7 +15,7 @@
    public void m_6800_(long p_128923_) {
       this.f_128919_ += p_128923_ / 8L;
       if (this.f_128919_ > this.f_128918_) {
-         throw new RuntimeException("Tried to read NBT tag that was too big; tried to allocate: " + this.f_128919_ + "bytes where max allowed: " + this.f_128918_);
+         throw new RuntimeException("大きすぎるNBTタグを読み取ろうと、もしくは割り当てようとしました: " + this.f_128919_ + "最大許容バイト: " + this.f_128918_);
       }
    }
 }
