--- a/main/java/net/minecraft/advancements/TreeNodePosition.java
+++ b/main/java/net/minecraft/advancements/TreeNodePosition.java
@@ -20,7 +20,7 @@
 
    public TreeNodePosition(Advancement p_16567_, @Nullable TreeNodePosition p_16568_, @Nullable TreeNodePosition p_16569_, int p_16570_, int p_16571_) {
       if (p_16567_.m_138320_() == null) {
-         throw new IllegalArgumentException("Can't position an invisible advancement!");
+         throw new IllegalArgumentException("目に見えない進歩を配置することはできません！");
       } else {
          this.f_16554_ = p_16567_;
          this.f_16555_ = p_16568_;
@@ -213,7 +213,7 @@
 
    public static void m_16587_(Advancement p_16588_) {
       if (p_16588_.m_138320_() == null) {
-         throw new IllegalArgumentException("Can't position children of an invisible root!");
+         throw new IllegalArgumentException("見えない子ルートを配置することはできません！");
       } else {
          TreeNodePosition treenodeposition = new TreeNodePosition(p_16588_, (TreeNodePosition)null, (TreeNodePosition)null, 1, 0);
          treenodeposition.m_16572_();
