--- a/main/java/net/minecraft/server/rcon/thread/RconThread.java
+++ b/main/java/net/minecraft/server/rcon/thread/RconThread.java
@@ -46,7 +46,7 @@
                this.m_11618_();
             } catch (IOException ioexception) {
                if (this.f_11515_) {
-                  f_11601_.info("IO exception: ", (Throwable)ioexception);
+                  f_11601_.info("IO例外: ", (Throwable)ioexception);
                }
             }
          }
@@ -68,7 +68,7 @@
       if (0 < i && 65535 >= i) {
          String s1 = dedicatedserverproperties.f_139749_;
          if (s1.isEmpty()) {
-            f_11601_.warn("No rcon password set in server.properties, rcon disabled!");
+            f_11601_.warn("server.propertiesにrconパスワードが設定されておらず、rconが無効になっています!");
             return null;
          } else {
             try {
@@ -78,16 +78,16 @@
                if (!rconthread.m_7528_()) {
                   return null;
                } else {
-                  f_11601_.info("RCON running on {}:{}", s, i);
+                  f_11601_.info("{} で実行されているRCON:{}", s, i);
                   return rconthread;
                }
             } catch (IOException ioexception) {
-               f_11601_.warn("Unable to initialise RCON on {}:{}", s, i, ioexception);
+               f_11601_.warn("{} でRCONを初期化できません:{}", s, i, ioexception);
                return null;
             }
          }
       } else {
-         f_11601_.warn("Invalid rcon port {} found in server.properties, rcon disabled!", (int)i);
+         f_11601_.warn("server.propertiesで無効なrconポート{}が見つかりました。rconは無効です!", (int)i);
          return null;
       }
    }
@@ -112,7 +112,7 @@
       try {
          p_11614_.close();
       } catch (IOException ioexception) {
-         f_11601_.warn("Failed to close socket", (Throwable)ioexception);
+         f_11601_.warn("ソケットを閉じることができませんでした", (Throwable)ioexception);
       }
 
    }
