--- a/main/java/net/minecraft/resources/RegistryReadOps.java
+++ b/main/java/net/minecraft/resources/RegistryReadOps.java
@@ -96,9 +96,9 @@
       for(ResourceLocation resourcelocation : collection) {
          String s1 = resourcelocation.m_135815_();
          if (!s1.endsWith(".json")) {
-            f_135634_.warn("Skipping resource {} since it is not a json file", (Object)resourcelocation);
+            f_135634_.warn("jsonファイルではないため、リソース {} をスキップします", (Object)resourcelocation);
          } else if (!s1.startsWith(s)) {
-            f_135634_.warn("Skipping resource {} since it does not have a registry name prefix", (Object)resourcelocation);
+            f_135634_.warn("レジストリ名のプレフィックスがないため、リソース {} をスキップします", (Object)resourcelocation);
          } else {
             String s2 = s1.substring(s.length(), s1.length() - ".json".length());
             ResourceLocation resourcelocation1 = new ResourceLocation(resourcelocation.m_135827_(), s2);
@@ -123,7 +123,7 @@
          Supplier<E> supplier = Suppliers.memoize(() -> {
             E e = p_135691_.m_6246_(resourcekey);
             if (e == null) {
-               throw new RuntimeException("Error during recursive registry parsing, element resolved too early: " + resourcekey);
+               throw new RuntimeException("再帰的なレジストリ解析中にエラーが発生しました。要素の解決が早すぎます: " + resourcekey);
             } else {
                return e;
             }
