--- a/main/java/net/minecraft/locale/Language.java
+++ b/main/java/net/minecraft/locale/Language.java
@@ -28,10 +28,10 @@
       Builder<String, String> builder = ImmutableMap.builder();
       BiConsumer<String, String> biconsumer = builder::put;
 
-      try (InputStream inputstream = Language.class.getResourceAsStream("/assets/minecraft/lang/en_us.json")) {
+      try (InputStream inputstream = Language.class.getResourceAsStream("/assets/minecraft/lang/ja_jp.json")) {
          m_128108_(inputstream, biconsumer);
       } catch (JsonParseException | IOException ioexception) {
-         f_128101_.error("Couldn't read strings from /assets/minecraft/lang/en_us.json", (Throwable)ioexception);
+         f_128101_.error("Couldn't read strings from /assets/minecraft/lang/ja_jp.json", (Throwable)ioexception);
       }
 
       final Map<String, String> map = builder.build();
