--- a/main/java/net/minecraft/advancements/critereon/EnterBlockTrigger.java
+++ b/main/java/net/minecraft/advancements/critereon/EnterBlockTrigger.java
@@ -22,7 +22,7 @@
       StatePropertiesPredicate statepropertiespredicate = StatePropertiesPredicate.m_67679_(p_31281_.get("state"));
       if (block != null) {
          statepropertiespredicate.m_67672_(block.m_49965_(), (p_31274_) -> {
-            throw new JsonSyntaxException("Block " + block + " has no property " + p_31274_);
+            throw new JsonSyntaxException("ブロック \"" + block + "\" のプロパティがありません" + p_31274_);
          });
       }
 
@@ -34,7 +34,7 @@
       if (p_31279_.has("block")) {
          ResourceLocation resourcelocation = new ResourceLocation(GsonHelper.m_13906_(p_31279_, "block"));
          return Registry.f_122824_.m_6612_(resourcelocation).orElseThrow(() -> {
-            return new JsonSyntaxException("Unknown block type '" + resourcelocation + "'");
+            return new JsonSyntaxException("不明なブロックの種類です '" + resourcelocation + "'");
          });
       } else {
          return null;
