--- a/main/java/net/minecraft/advancements/critereon/ItemPredicate.java
+++ b/main/java/net/minecraft/advancements/critereon/ItemPredicate.java
@@ -108,14 +108,14 @@
          MinMaxBounds.Ints minmaxbounds$ints = MinMaxBounds.Ints.m_55373_(jsonobject.get("count"));
          MinMaxBounds.Ints minmaxbounds$ints1 = MinMaxBounds.Ints.m_55373_(jsonobject.get("durability"));
          if (jsonobject.has("data")) {
-            throw new JsonParseException("Disallowed data tag found");
+            throw new JsonParseException("許可されていないデータタグが見つかりました");
          } else {
             NbtPredicate nbtpredicate = NbtPredicate.m_57481_(jsonobject.get("nbt"));
             Item item = null;
             if (jsonobject.has("item")) {
                ResourceLocation resourcelocation = new ResourceLocation(GsonHelper.m_13906_(jsonobject, "item"));
                item = Registry.f_122827_.m_6612_(resourcelocation).orElseThrow(() -> {
-                  return new JsonSyntaxException("Unknown item id '" + resourcelocation + "'");
+                  return new JsonSyntaxException("不明なアイテムIDです: '" + resourcelocation + "'");
                });
             }
 
@@ -124,7 +124,7 @@
                ResourceLocation resourcelocation1 = new ResourceLocation(GsonHelper.m_13906_(jsonobject, "tag"));
                tag = SerializationTags.m_13199_().m_5881_().m_13404_(resourcelocation1);
                if (tag == null) {
-                  throw new JsonSyntaxException("Unknown item tag '" + resourcelocation1 + "'");
+                  throw new JsonSyntaxException("不明なアイテムタグです: '" + resourcelocation1 + "'");
                }
             }
 
@@ -132,7 +132,7 @@
             if (jsonobject.has("potion")) {
                ResourceLocation resourcelocation2 = new ResourceLocation(GsonHelper.m_13906_(jsonobject, "potion"));
                potion = Registry.f_122828_.m_6612_(resourcelocation2).orElseThrow(() -> {
-                  return new JsonSyntaxException("Unknown potion '" + resourcelocation2 + "'");
+                  return new JsonSyntaxException("不明なポーションです: '" + resourcelocation2 + "'");
                });
             }
 
